include/RMGInnerBremsstrahlungProcess.hh

```diff
@@ -0,0 +1,131 @@
+#ifndef RMG_INNER_BREMSSTRAHLUNG_PROCESS_HH
+#define RMG_INNER_BREMSSTRAHLUNG_PROCESS_HH
+
+#include "G4ParticleDefinition.hh"
+#include "G4Positron.hh"
+#include "G4Step.hh"
+#include "G4SystemOfUnits.hh"
```

Comment:
please do not include `G4SystemOfUnits.hh` directly, but use something like `using u = CLHEP;`; you can then use all units with `u::keV` or similar.

This is probably the reason of the compile error here on CI. on CI we enforce that all code must compile without warnings.